opium_kernel/src/middlewares/logger.ml

```diff
@@ -29,29 +29,30 @@ let respond ?time_f handler req =
     let span, response_lwt = time_f f in
     response_lwt
     >|= fun response ->
-    let code = response.Response.status |> Httpaf.Status.to_string in
-    Log.info (fun m -> m "Responded with HTTP code %s in %a" code Mtime.Span.pp span);
-    Log.debug (fun m -> m "%a" Response.pp_http response);
+    let code = response.Response.status |> Status.to_string in
+    Logs.info ~src:log_src (fun m ->
```

Comment:
This change from `Log.info` to `Logs.info ~src:log_src` is not required. module Log is defined as

```ocaml
let log_src = Logs.Src.create "opium.server"

module Log = (val Logs.src_log log_src : Logs.LOG)
```
All logging functions in Log are for the source log_src